testindex/__init__.py

```diff
@@ -4,21 +4,14 @@
 
 
 def get_parser():
+    """Initializes and returns an argument parser object"""
     parser = ArgumentParser(description="This script is used to validate the contents on the index.d directory.")
 
-    parser.add_argument("-d",
-                        "--dump_directory",
-                        help="Specify a custom data dump directory",
-                        metavar='DUMPDIR',
-                        nargs=1,
-                        action="store")
-
     parser.add_argument("-i",
-                        "--indexd_location",
-                        help="Specify location of indexd. Default is ./index.d",
-                        metavar="INDEXD",
-                        nargs=1,
-                        action="store")
+                        "--index",
+                        help="Specify location of index. Default is ./index.d",
+                        metavar="INDEX_PATH",
+                        default="./index.d")
```

Comment:
For the sake of readability (because that's quite important in Python), can we have something like:

```python
parser.add_argument(
    "-i",
    "--index",
    help="Specify location of index. Default is ./index.d",
    metavar="INDEX_PATH",
    default="./index.d"
)
```